lib/loader.js

```diff
@@ -30,8 +39,8 @@ module.exports = function(content, map) {
 
 	processCss(content, map, {
 		mode: moduleMode ? "local" : "global",
-		from: loaderUtils.getRemainingRequest(this),
-		to: loaderUtils.getCurrentRequest(this),
+		from: loaderUtils.getRemainingRequest(this).split("!").pop(),
```

Comment:
@michael-ciniawsky we can have `remainingRequest` https://github.com/webpack/loader-utils/blob/master/lib/getRemainingRequest.js#L4